common/views/components/Tags/Tags.js

```diff
@@ -33,10 +33,10 @@ const Tags = ({ tags }: Props) => {
           [spacing({ s: 0 }, { padding: ['left'], margin: ['top'] })]: true,
         })}
       >
-        {tags.map(({ textParts, linkAttributes }) => {
+        {tags.map(({ textParts, linkAttributes }, i) => {
           return (
             <li
-              key={textParts.join('-')}
+              key={textParts.concat(i).join('-')}
```

Comment:
Why not just use `i` and leave a note why we can't do something else?
Or we could send a key down the pipes and attach the type of tag it is ie.. `lcsh` or `mesh`.